README.md

```diff
@@ -211,9 +211,31 @@ ipyreact.ValueWidget(
 )
 ```
 
+### Importing ReactJS based libraries
+
+Imported libraries must require/import `react` and `react-dom` packages provided by `ipyreact`, otherwise things will break (see [FAQ](#faq)). 
```

Comment:
```suggestion
Imported libraries must use the `react` and `react-dom` packages provided by `ipyreact`, otherwise things will break (see [FAQ](#faq)). 
```